app/helpers/assignment_index_helper.rb

```diff
@@ -7,17 +7,17 @@ def cal_day(day:)
 
   def cal_assignment(assignment:, day:)
     return if assignment.blank?
-    ass = 'assignment'
-    ass += '-user' if assignment.user == @current_user
+    assign = 'assignment'
+    assign += '-user' if assignment.user == @current_user
 
     if day == assignment.start_date && day == assignment.end_date
```

Comment:
Why can't you just say `day == assignment.start_date == assignment.end_date`? Maybe that's more of a mess, but it would remove the need to write `day` twice. Idk. Maybe it's fine.